### PR TITLE
Correct work without view context in helpers

### PR DESCRIPTION
**Pull request by alexeyfrank**

### CODE REVIEW

File: lib/webpack/configuration.rb [hunk 1]
```diff
@@ -6,6 +6,9 @@ class Configuration
     # @attribute host [String]
     attr_accessor :host
 
+    # @attribute host [String]
```
Comment (ertrzyiks) [inline lib/webpack/configuration.rb]: Please use the correct attribute name here

**Comment by alexeyfrank** [comment]
@ertrzyiks fixed

**Comment by irvingwashington** [comment]
I'm not very familiar with the codebase, but LGTM 👍

**Comment by ertrzyiks** [comment]
👍

**Pull request merged**